app/javascript/react/components/App.js

```diff
@@ -10,10 +11,10 @@ export const App = () => {
       <Switch>
         <Route exact path="/" component={ParksIndexContainer}/>
         <Route exact path="/parks" component={ParksIndexContainer}/>
+        <Route exact path="/parks/:id" component={ParksShowContainer}/>
```

Comment:
dynamic paths should be last!

Reply:
Thanks! We moved the /new route up to address this.